bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/dto/Model.kt

```diff
@@ -23,10 +25,48 @@ enum class BtcNetwork(val params: NetworkParameters) {
 
 open class Conflictable(var errorCode: String? = null, var message: String? = null)
 
+
+data class MasterContractsRequest(
+    @NotNull val network: EthereumNetworkProperties? = null,
```

Comment:
Strange combination of `@NotNull` and `EthereumNetworkProperties? = null`. Is everything ok?

Reply:
Yes, Because @NotNull is for API call, and null is for default constructor, which used by Spring for serialization.